contents/staticAnalysis/ESLint.md

```diff
@@ -0,0 +1,130 @@
+# ESLint
+
+Author: [Nicholas Chua](https://github.com/nus-cs3281/2018/blob/master/students/nicholasChua/nicholasChua-Resume.md)
+
+## Overview
+
+[ESLint](https://eslint.org/docs/about/) is an open source JavaScript linter.
+
+Code linting is a type of static analysis that helps find problematic patterns or code that does not follow a certain style guideline.
+
+JavaScript, being a loosely-typed, interpreted language, is prone to developer errors and therefore linting is especially important when working with JavaScript.
+
+## Features
+
+### In Built Rules
+[In built rules](https://eslint.org/docs/rules/) in ESLint are split into a few categories:
+
+* Possible Errors
+* Best Practices
+* Variables
+* Strict Mode
+* Node.js and CommonJS
+* Stylistic Issues
+* ECMAScript 6 (ES6)
+
+### Custom Rules
+Custom rules can be imported via Plugins. You can [learn how to write your own](https://eslint.org/docs/developer-guide/working-with-plugins) or [use existing ones](https://www.npmjs.com/browse/keyword/eslintplugin) that are already published to npm.
+
+### Integrations
+ESLint has [Editor Integrations](https://eslint.org/docs/user-guide/integrations#editors) such as error highlighting in many popular editors.
+
+## How to use it
+
+### Installation
+ESLint requires Node.js (>= 4.x) and npm version 2+. You can [install it with npm](https://eslint.org/docs/user-guide/getting-started#installation-and-usage).

```

Comment:
As this is not meant to be a tutorial, you can cut down on this type of details. It's good enough to mention that ESLint can be installed via npm and point to a detailed installation guide.